ExpressWash/ExpressWash/CarController.swift

```diff
@@ -34,7 +34,6 @@ class CarController {
                 return
             }
 
-            UserController.shared.sessionUser?.addToCars(car)
```

Comment:
I see... you took this out because it's implemented in the completion. Good call. That's a better place for it.